tests/main/security-device-cgroups-helper/task.yaml

```diff
@@ -25,6 +25,9 @@ execute: |
     #shellcheck source=tests/lib/systems.sh
     . "$TESTSLIB/systems.sh"
     libexecdir=$(os.paths libexec-dir)
+    if tests.info is-snapd-pkg-repo; then
+        libexecdir="/snap/snapd/current$libexecdir"
```

Comment:
I missed this earlier but this test is no longer realistic.

Snapd generates udev rules that call the distribution-packaged snap-device-helper and as we are painfully aware of, that helper does not re-execute. If this test needs this sort of adjustment to pass then the test is measuring something that doesn't really work everywhere and needs to correctly be skipped if the host version of snap-device-helper is just too old.

Reply:
This isn't a full e2e snapd test, bur rather we pull out one of the bits relevant in the sandbox setup, rig the configuration in a way that results in a test environment and check that the cgroup filtering rules are what we expect them to be. In essence we need to use the helper from snap, as it's the only location which is now carrying the potentially modified code (since snapd package is installed from distribution repos and so it contains older, already merged code).

Reply:
comment added in the test

Reply:
Thanks, this is good now.